tcms/testcases/tests/test_models.py

```diff
@@ -157,3 +159,18 @@ def test_send_mail_to_case_author(self, send_mail):
             recipients,
             fail_silently=False,
         )
+
+class TestActualDurationProperty(BasePlanCase):
+    """Test TestExecution.actual_duration"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super(TestActualDurationProperty, cls).setUpTestData()
+
+        cls.test_execution = TestExecution()
+        cls.test_execution.start_date = timezone.today()
+        cls.test_execution.stop_date = timezone.now()
```

Comment:
Use `django.utils.timezone.now` + a `timedelta` or hardcode the values of the `datetime` objects like you did before but don't parse them from string.